Lib/pyclbr.py

```diff
@@ -284,6 +320,7 @@ def _readmodule(module, path, inpackage=None):
     f.close()
     return dict
 
+
```

Comment:
Sorry about these.  I have the linter turned on in my editor and it flags these.  I need to figure out how to bring up a diff (like what github shows) to see the before and after code to know if it's a change I made or if it was how the code was before I touched it.